### PR TITLE
fix: MD007/ul-indent

### PR DESCRIPTION
**Pull request by nschonni**
- Unordered list indentation
- Added code fence languages
- Trim trailing/leading spaces

### CODE REVIEW

File: docs/framework/data/wcf/streaming-provider-wcf-data-services.md [hunk 1]
```diff
@@ -1,124 +1,136 @@
...
+|<xref:System.Data.Services.Providers.IDataServiceStreamProvider.ResolveType%2A>|Returns a namespace-qualified type name that represents the type that the data service runtime must create for the media link entry that is associated with the data stream for the media resource that is being inserted.|
+
+## Creating the Streaming Data Service
+
+To provide the [!INCLUDE[ssAstoria](../../../../includes/ssastoria-md.md)] runtime with access to the <xref:System.Data.Services.Providers.IDataServiceStreamProvider> implementation, the data service that you create must also implement the <xref:System.IServiceProvider> interface. The following example shows how to implement the <xref:System.IServiceProvider.GetService%2A> method to return an instance of the `PhotoServiceStreamProvider` class that implements <xref:System.Data.Services.Providers.IDataServiceStreamProvider>.
+
+[!code-csharp[Astoria Photo Streaming Service#PhotoServiceStreamingProvider](../../../../samples/snippets/csharp/VS_Snippets_Misc/astoria%20photo%20streaming%20service/cs/photodata.svc.cs#photoservicestreamingprovider)]
```
Comment (BillWagner) [inline docs/framework/data/wcf/streaming-provider-wcf-data-services.md]: I'm a bit confused.  Shouldn't the space characters change to `_` to match dotnet/samples#810?  

And, that begs the question: will other docs need to be updated for those folder and file name changes?

Comment (BillWagner) [reply docs/framework/data/wcf/streaming-provider-wcf-data-services.md]: I just saw #11784 

If I understand correctly, we should merge this in the following order:
#11783 
dotnet/samples#810
#11784 

Does that make sense @nschonni ?

Comment (nschonni) [reply docs/framework/data/wcf/streaming-provider-wcf-data-services.md]: Thanks, I thought I had undone all of these to handle them separately. I've rebased these ones out so those other PRs should be independent again

**Comment by nschonni** [comment]
@BillWagner I've rebased this now that the other PRs landed

**Comment by nschonni** [comment]
@BillWagner anything to change here?

**Comment by BillWagner** [comment]
@nschonni Thanks for the ping. I was OOF for a long weekend, and this dropped off my radar. Reviewing now.

**Review by BillWagner — APPROVED** [review]
Thanks again for all the work @nschonni 

This is great. I'll :shipit: now.

**Comment by nschonni** [comment]
@BillWagner not a problem

**Pull request merged**